core/src/main/java/org/geysermc/geyser/command/defaults/ConnectionTestCommand.java

```diff
@@ -124,6 +149,10 @@ public void execute(@Nullable GeyserSession session, GeyserCommandSource sender,
                 sendLinks(sender);
             } catch (Exception e) {
                 sender.sendMessage("Error while trying to check your connection!");
+                if (e instanceof FileNotFoundException) {
+                    sender.sendMessage("Specify a valid IP or domain to check!");
+                    return;
+                }
                 geyser.getLogger().error("Error while trying to check your connection!", e);
             }
```

Comment:
do we still wanna log orrrrr

Reply:
i wouldnt; maybe at debug level
My point is to reduce the amount of errors (since people think geyser malfunctioned), even though trash might be inputted

Reply:
I would not log if this is the case just let the sender know, but also this should be in a seperate catch (assuming Java lets you do that)
EG:
```java
} catch (FileNotFoundException e) {
    .....
} catch (Exception e) {
    ....
}
```